src/tests/tracing/eventcounter/runtimecounters.csproj

```diff
@@ -4,7 +4,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <!-- Tracing tests routinely time out with jitstress and gcstress -->
     <GCStressIncompatible>true</GCStressIncompatible>
-    <JitOptimizationSensitive>true</JitOptimizationSensitive>
+    <JitOptimizationSensitive Condition="'$(TestBuildMode)' != 'nativeaot'">true</JitOptimizationSensitive>
```

Comment:
Is this just because we don't want the environment variable check since in never matters for nativeaot? If so, can we just update the infrastructure to not add the check for nativeaot (similar to how it is skipped for mono)?

Reply:
Oops missed this comment, sorry! yep, it makes sense to change the check in infrastructure, will push a change for this

Reply:
#86523